src/openforms/formio/dynamic_config/__init__.py

```diff
@@ -44,7 +45,25 @@ def rewrite_formio_components_for_request(
     Loop over the formio configuration and inject request-specific configuration.
     """
     for component in configuration_wrapper:
-        register.update_config_for_request(component, request=request)
+        # prevent multiple overwrites of content components (bandaid for #2895)
+        # postprocess (a) non-content components, (b) components with inline style
+        # (c) components with nonce that does not match the nonce on the request
+
+        # get HTML (if any) from component and parse it
+        if html_string := component.get("html", None):
+            request_nonce = request.headers.get("X-Csp-Nonce", None)
```

Comment:
You can use `NONCE_HTTP_HEADER` from `src/csp_post_processor/constants.py`, and the `.get` returns `None` by default already so you can leave the second argument off